tests/create_me/flipper.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "0.2.1",
+  "name": "flipper",
+  "instructions": [
+    {
+      "name": "new",
+      "docs": [
```

Comment:
Remove this (should probably be in `.gitignore` or temp directory

Reply:
added to gitignore